programs/leap-util/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_executable( leap-util main.cpp actions/generic.cpp actions/blocklog.cpp actions/snapshot.cpp actions/chain.cpp)
+
+if( UNIX AND NOT APPLE )
+  set(rt_library rt )
+endif()
+
+target_include_directories(leap-util PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+
+target_link_libraries( leap-util
+        PRIVATE appbase version
+        PRIVATE eosio_chain chain_plugin fc leap-cli11 producer_plugin ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
+
+copy_bin( leap-util )
+install( TARGETS
+   leap-util
+
+   COMPONENT base
```

Comment:
I'm not sure this is in the right location for `COMPONENT`, does it work like this? Something similar to nodeos' cmake `install()` is probably all that is required here since it's only `RUNTIME` I think

Reply:
Thank you! Changed.